Standards/scs-0102-v2-image-metadata.md

```diff
@@ -0,0 +1,240 @@
+the field `hotfix_hours` is optional.
+
+All dates are in UTC.
+
+#### Example
+
+Providing an image with name `OPSYS MAJ.MIN` with
+`replace_frequency=monthly`, `provided_until=2022-09-30`, `uuid_validity=2022-12-31`,
+`hotfix_hours=0`
+means that we will have a new image with this name at least once per month (starting from
+the initial release) until the end of September 2022. Old images will be hidden and/or
+renamed, but remain accessible via their UUID until at least the end of 2022 (in Universal Time).
+The provider makes an effort to replace images upon critical security issues out of order.
+
+### Image Origin
+
+- Mandatory: `image_source` needs to be a URL to point to a place from which the image can be downloaded.
+  (Note: This may be set to the string "private" to indicate that the image can not be freely
+  downloaded.)
+- Mandatory: `image_description` needs to be an URL (or text) with release notes and other human readable
+  data about the image.
+
+- Recommended _tag_: `managed_by_VENDOR`
+
+Note that for most images that come straight from an upstream source, `image_description` should point
+to a an upstream web page where these images are described. If download links are available as well
+on that page, `image_source` can point to the same page, otherwise a more direct link to the image
+should be used, e.g. directly linking the `.qcow2` or `.img` file.
+If providers have their own image building machinery or do some post-processing on top of
+upstream images, they should point to the place where they document and offer these images.
+
+### Image build info
+
+- Mandatory: `image_build_date` needs to be `YYYY-MM-DD` or `YYYY-MM-DD hh:mm[:ss]` (time in UTC,
+  24hrs clock).
+  All publicly released and generally recommended patches before this date must be included in the
+  image build. If the cutoff date is earlier, this cutoff date needs to be set instead, even
+  if the actual build happens significantly after the cutoff date. If not all patches can be
+  included for a good reason, then the `patchlevel` field (see below) must be used to describe
+  the patch status.
+- Mandatory: `image_original_user` is the default login user for the operating system which can connect
+  to the image via the injected SSH key or provided password. (This can be set to `none` if no default
+  user name exists for the operating system.)
+- Optional: `patchlevel` can be set to an operating specific patch level that describes the
+  patch status — typically we would expect the `image_build_date` to be sufficient.
+
+- Recommended: `os_hash_algo` and `os_hash_value`: The sha256 or sha512 hash
+  for the image file. (This references the image file in the format it is stored in, we
+  recommend raw over qcow2 for systems that use ceph.) Note that these values are
+  typically generated automatically upon image registration.
+- Recommended: Digital image signature according to the [glance image
```

Comment:
Note: this is not possible for Public images and thus should not be stated here.
I have an example to show here: https://input.scs.community/RID5cVHpTaG_AEDYsB9gVQ

Reply:
Probably a very good point, but I suppose this PR is the wrong place, because it only splits the standard into two separate documents.

Reply:
@josephineSei To be even more clear: I didn't write any of that. I only copied the text into a new file. However, the new file will have draft status (as is required by scs-0001), so we can still discuss this topic and change the document once this PR is through. By any means, raise your concerns, but preferably not with this PR (but rather in the team IaaS meetings), because the problem exists in V1 of the standard, which is currently in effect, and V2 will be in draft state afterwards and can be changed. This course of action had actually been agreed upon with Markus.